kroxylicious-operator/src/test/resources/DerivedResourcesTest/two-filters/out-Deployment-example.yaml

```diff
@@ -73,10 +73,21 @@ spec:
             - mountPath: "/opt/kroxylicious/secure/secret/my-secret"
               name: "secrets-my-secret"
               readOnly: true
+            - mountPath: "/opt/kroxylicious/cluster/foo/target/tls-key"
+              name: "vc-target-tls-foo"
+              readOnly: true
```

Comment:
Note that the mount paths and volume names are constructed to avoid collidting, even if a filter references the same secret as the target cluster. I eventually concluded that could simplify things if we gradually roll out support for dynamic cert update. That has potential to be quite a complicated story because the operator doesn't really know whether an interpolated secret reference in a filter CR is going to support dynamic update.